tests/routes/metadata_test.py

```diff
@@ -4,6 +4,18 @@
 
 
 class MetadataTest(FunctionalTestBase):
+    def test_head_metadata(self):
+        self.route_tester \
+            .metadata() \
+            .route_params(bucket_name="test", path="test") \
+            .expect(200, headers={
+                "Link": [
+                    "/test/artifact/test; rel=self; title=artifact",
+                    "/test/artifact/test/_meta; rel=related; title=metadata"
```

Comment:
I think these are wrong.  _meta should be self, the artifact should be "related"